src/main/fc/settings.yaml

```diff
@@ -64,7 +64,7 @@ tables:
   - name: nav_rth_alt_mode
     values: ["CURRENT", "EXTRA", "FIXED", "MAX", "AT_LEAST", "AT_LEAST_LINEAR_DESCENT"]
   - name: osd_unit
-    values: ["IMPERIAL", "METRIC", "UK"]
+    values: ["IMPERIAL", "METRIC", "METRIC MPH", "UK"]
```

Comment:
Doesn't this need to be `METRIC_MPH` (with underscore)? If not for functionality then for consistency with the other settings?

Reply:
You're right. I doubt it would work in CLI without the underscore. I'm on it.